services/user.py

```diff
@@ -0,0 +1,55 @@
+from typing import Optional
+
+from django.contrib.auth.hashers import make_password
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: Optional[str] = None,
+        first_name: Optional[str] = None,
+        last_name: Optional[str] = None
+) -> None:
+    hashed_password = make_password(password)
+    user = User.objects.create_user(

```

Comment:
Use get_user_model() instead of User. Check out the [checklist](https://github.com/mate-academy/py-movie-moviesession-cinemahall/blob/master/checklist.md#code-efficiency).

Reply:
I dunno which of these two options is more appropriate:
User = get_user_model()
user = User.objects...
or
user = get_user_model().objects...
:)